variables.tf

```diff
@@ -97,7 +113,7 @@ variable "esxi_host_count" {
 }
 
 variable "vcenter_portgroup_name" {
-  default = "VM Private Net 1"
+  default = "VM Public Net 1"
```

Comment:
This can be overridden to Public in the Anthos module. 

I don't know the ramifications of changing this default (no description on the field). I would assume that the Private default is the more secure default.

Reply:
The default to the Public Net will make it so we don't need a VPN for vCenter. It will make a lot of folks lives easier if they are just testing.

Reply:
That seems like a win.

Can you add a description about what the choices are and what the benefits/side-effects of this value while you are here.

Reply:
(this is a change we will want to highlight in the changelog for this vsphere module)

Reply:
I added a more verbose comment for this. It may be too verbose. But it's there.